pkg/operator/connectivitycheckcontroller/connectivity_check_controller.go

```diff
@@ -77,6 +87,18 @@ func (c *connectivityCheckController) Sync(ctx context.Context, syncContext fact
 		return nil
 	}
 
+	enabled, err := Enabled(operatorSpec)
+	if err != nil {
+		return err
+	}
+
+	if !enabled {
+		// TODO delete CRD
+		return nil
+	}
+
+	// TODO ensure CRD exists
```

Comment:
This I could actually let you leave as a TODO, but if you're going to the do the above.

Reply:
Done